src/edu/csus/ecs/pc2/core/scoring/NewScoringAlgorithm.java

```diff
@@ -519,7 +527,10 @@ private IMemento addTeamMemento(IMemento mementoRoot, IInternalContest contest,
         standingsRecordMemento.putInteger("rank", standingsRecord.getRankNumber());
         standingsRecordMemento.putInteger("index", indexNumber);
         Account account = contest.getAccount(standingsRecord.getClientId());
-        standingsRecordMemento.putString("teamName", account.getDisplayName());
+ 
+        standingsRecordMemento.putString("teamName", ScoreboardVariableReplacer.substituteDisplayNameVariables(teamVarDisplayString, account, group));
+//        standingsRecordMemento.putString("teamName", account.getDisplayName());
```

Comment:
Should the commented out old line be removed?  it is always available from git, and it clutters the code.

Reply:
removed, pushed fix.